build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
         classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:5.1.2'
         
         // Scala plugins
-        classpath "com.adtran:scala-multiversion-plugin:1.0.35"
+        classpath "com.adtran:scala-multiversion-plugin:1.+"
```

Comment:
Updated the classpath as per: https://github.com/ADTRAN/gradle-scala-multiversion-plugin#scala-multi-version-plugin

Reply:
I'm not sure why to do this.  Wouldn't it preferable to have repeatable builds by continuing to pin to a specific version?

Reply:
I think its to catch any bug fixes in the plugin, especially with regards with newer versions of gradle.

Reply:
Fixed to latest version.

Reply:
Weird that at https://github.com/ADTRAN/gradle-scala-multiversion-plugin/releases still shows 1.0.35 as the latest release.

Reply:
https://plugins.gradle.org/plugin/com.adtran.scala-multiversion-plugin

Reply:
Yeah, I wonder why the discrepancy. Anyway, no worries.